run-bungeecord.sh

```diff
@@ -2,7 +2,9 @@
 
 : ${TYPE:=BUNGEECORD}
 : ${MEMORY:=512m}
-BUNGEE_HOME=/server
+BUNGEE_HOME=./server
```

Comment:
Was this change needed? It is supposed to line up with

https://github.com/itzg/docker-bungeecord/blob/master/Dockerfile#L4

Reply:
Oh, that its misstake i have change the path for my test, sorry